libs/domain/cart/list-item/list-item.component.ts

```diff
@@ -0,0 +1,137 @@
+import { CartComponentMixin, CartService, PriceMode } from '@spryker-oryx/cart';
+import { resolve } from '@spryker-oryx/di';
+import { AlertType } from '@spryker-oryx/ui';
+import { I18nMixin, Size } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { CartListItemProperties } from './list-item.model';
+import { cartListItemStyles } from './list-item.styles';
+
+const size = Size.Md;
+
+export class CartListItemComponent
+  extends CartComponentMixin(I18nMixin(LitElement))
+  implements CartListItemProperties
+{
+  static styles = cartListItemStyles;
+
+  protected cartService = resolve(CartService);
+
+  @property({ type: Boolean, reflect: true }) open?: boolean;
+
+  protected setDefault(): void {
+    this.cartService
+      .updateCart({
+        cartId: this.$cart()?.id,
+        isDefault: true,
+      })
+      .subscribe();
```

Comment:
subscribe is not needed (unless you want to react on error, or wait for the successuf execution)